prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -274,16 +274,21 @@ def __get_users__(self):
                     if not self.audit_resources or (
                         is_resource_filtered(user["Arn"], self.audit_resources)
                     ):
-                        if "PasswordLastUsed" not in user:
-                            users.append(User(name=user["UserName"], arn=user["Arn"]))
-                        else:
-                            users.append(
-                                User(
-                                    name=user["UserName"],
-                                    arn=user["Arn"],
-                                    password_last_used=user["PasswordLastUsed"],
-                                )
+                        try:
+                            user_login_profile = self.client.get_login_profile(
+                                UserName=user["UserName"]
                             )
+                        except self.client.exceptions.NoSuchEntityException:
+                            user_login_profile = None
+
```

Comment:
Please add `except Exception as error:` here to catch any other exception with a log level ERROR and setting the `user_login_profile` to `None`.

Reply:
Added 🫡